docs/developers/contributing.md

```diff
@@ -210,6 +217,11 @@ You can then make a
 
 ## Building the docs
 
+Install the documents tools:
+```sh
+pip install -e ".[doc]"  # (quotes only needed for zsh shell)
```

Comment:
Any more detail about zsh? I use zsh and do not need to use `"` when installing extras.

Reply:
I've always had to use quotes on my mac for zsh, so I think we should leave this. It's been reproduced from other instructions on the website so it's also consistent to keep it here

Reply:
I think the reason why you don't need to expand, @Czaki, is because you are using some zsh plugin that sets this alias (or you set it yourself):

```
alias pip="noglob pip"
```

Reply:
ok, understand. oh-my-zsh rulez.